### DISCUSSION TITLE
xcb plugin issue

### DISCUSSION THREAD

**Discussion by marcelbenhur** [lxqt/lxqt#2158] [Q&A]
Hello,
I launch a new software (onionshare in thas case). It doesn't start and i've the following message
--------------------------------------------------------------------------
qt.qpa.xcb: could not connect to display :0
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, webgl, xcb.

Aborted (core dumped)
---------------------------------------------------------------------------
could you help me.
Regards

**Comment by tsujan** [comment]
Why not reporting the problem to the developers of the app in question, its package maintainers or its forum (if any)?

**Reply by marcelbenhur** [reply]
This app works perfectly on other linux machines (ubuntu, debian, raspbian included lubuntu). I guess it's a  bad configuration of LXQT on this specific  lubuntu machine. It's why i post on this forum.
